extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -993,6 +1049,7 @@ private static String getVideoInfoUrl(final String id) {
                     if (itagItem.itagType == itagTypeWanted) {
                         // Ignore streams that are delivered using YouTube's OTF format,
                         // as those only work with DASH and not with progressive HTTP.
+                        // Those should be extracted when parsing the DASH manifest.
```

Comment:
Can't you parse these OTF streams instead of making another network request to get the DASH manifest?

Reply:
> - First, we don't know how many segments there are in the streams without fetching the stream url, using a head or a post request (the first segment/sequence has more headers returned)
> - Then most of times, stream sequences are 5 seconds long, but it may change sometimes, depending of streams. This information is returned for each sequence in the DASH manifest
> - Finally, how clients will be able to read these streams? Is ExoPlayer able natively to read this stream type? I don't think so and I think Google implemented an extension in their players (Android, Web, iOS, ...) to read natively this stream type.
> 
> So I don't what to anwser for your question.
> > - Then most of times, stream sequences are 5 seconds long, but it may change sometimes, depending of streams. This information is returned for each sequence in the DASH manifest
> 
> I just saw a video which has 10 seconds sequences.
> But in this video, some streams have also 5 seconds sequences!
> And also (always for this video), the DASH manifest returned for the Android client has less streams than the one returned for the web clients!

@TiA4f8R mmmh ok then, let's stick to parsing the DASH manifest for now, and eventually look deeper into it when switching to the mobile API